tui/tuning.sh

```diff
@@ -25,8 +25,12 @@ for option in "${available_options[@]}"; do
 done
 
 TUNING=$(whiptail "${whiptail_args[@]}" 3>&1 1>&2 2>&3)
-export TUNING
 
-if [ -z "$TUNING" ]; then
-  exit 0
-fi
+exit_status=$?
+
+if [ $exit_status == 0 ]; then
```

Comment:
The `"` are missing around `$exist_status` and `-eq` should be used instead of `==`.